biz.aQute.resolve/test/biz/aQute/resolve/obr/plugin/ObrExporterTest.java

```diff
@@ -0,0 +1,149 @@
+package biz.aQute.resolve.obr.plugin;
+
+import static aQute.bnd.build.Workspace.findWorkspace;
+import static aQute.lib.io.IO.getFile;
+import static biz.aQute.resolve.Bndrun.createBndrun;
+import static biz.aQute.resolve.obr.plugin.ObrExporter.TYPE;
+import static java.util.zip.GZIPInputStream.GZIP_MAGIC;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import aQute.bnd.osgi.repository.XMLResourceParser;
+import aQute.lib.io.IO;
+import biz.aQute.resolve.Bndrun;
+
+public final class ObrExporterTest {
+
+	private static Bndrun		project;
+	private static ObrExporter	underTest;
+
+	@BeforeClass
+	public static void setup() throws Exception {
+           project = createBndrun(findWorkspace(IO.work), getFile("test.bndrun"));
```

Comment:
You don't want to use static setup and teardown since there is the problem of side effects of one test method affecting other test methods. 

Also, you do not want to use the actual Bnd workspace as the workspace for your test. You need to construct a specific workspace in testdata to use. And then copy this workspace into the test output folder and use it from there. This also prevents side effects from one test method from influencing other test methods.

Reply:
Yeah, you are right. Will move to method level setup and teardown.

Reply:
DONE ✔️